src/containers/collaborations/components/AddOrganizationsToCollaborationModal.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * @flow
+ */
+
+import React, { useMemo, useState } from 'react';
+
+import { List, Map } from 'immutable';
+import { CollaborationsApiActions } from 'lattice-sagas';
+import { ActionModal, Label, Select } from 'lattice-ui-kit';
+import { useRequestState } from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import { RequestStates } from 'redux-reqseq';
+import type { Organization, UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import { ModalBody, ResetOnUnmount, StackGrid } from '../../../components';
+import { COLLABORATIONS } from '../../../core/redux/constants';
+import { selectOrganizations } from '../../../core/redux/selectors';
+import type { ReactSelectOption } from '../../../types';
+
+const { ADD_ORGANIZATIONS_TO_COLLABORATION, addOrganizationsToCollaboration } = CollaborationsApiActions;
+
+const RESET_ACTIONS = [ADD_ORGANIZATIONS_TO_COLLABORATION];
+
+const AddOrganizationsToCollaborationModal = ({
+  collaborationId,
+  isVisible,
+  onClose,
+  participatingOrganizations
```

Comment:
No change required, just thinking out loud. Did you consider passing the collaboration object itself instead of the id and orgs separately? If yes, why did you do it this way? 

Passing `participatingOrganizations` opens the possibility for the wrong value to be passed, even though that's pretty unlikely. Furthermore, you could alternatively just pass the collab id and then `selectCollaboration` to get the collab object and then get the org ids.

Reply:
I think that I originally just passed the collaborationId, and then realized I needed the participating orgs and didn't think to just pass the collaboration object or select the collaboration. I can update.